readthedocs/doc_builder/templates/doc_builder/conf.py.tmpl

```diff
@@ -76,7 +76,7 @@ context = {
     'current_version': "{{ version.verbose_name }}",
     'version_slug': "{{ version.slug }}",
     'MEDIA_URL': "{{ settings.MEDIA_URL }}",
-    'STATIC_URL': "{{ settings.STATIC_URL }}",
+    'STATIC_URL': "{{ project.get_static_path }}",
```

Comment:
I'm a little worried about this possibly breaking in production. We need to be really careful here, because there's likely weird edge cases given the historical usage.